src/pywink/devices/standard/__init__.py

```diff
@@ -129,13 +129,25 @@ def update_state(self, require_desired_state_fulfilled=False):
             if not is_desired_state_reached(power_strip[0]):
                 return
 
+    def _update_state_from_response(self, response_json, require_desired_state_fulfilled=False):
+        """
+        :param response_json: the json obj returned from query
+        :return:
+        """
+        if 'data' in response_json:
```

Comment:
I would prefer this function no to have to change to support pubnub.  See my other note in this file.